server/controllers/getArtist.js

```diff
@@ -0,0 +1,28 @@
+const { getArtistQuery } = require('../database/queries');
+
+const getArtist = async (req, res, next) => {
+  try {
+    const { artistId } = req.params;
+    if (artistId > 0) {
+      const { rows } = await getArtistQuery(artistId);
+      const data = { ...rows[0] };
```

Comment:
You can use destructuring to get object directly
```suggestion
       const {
        rows: [data],
      } = await getArtistQuery(artistId);
```